azure_functions_worker/dispatcher.py

```diff
@@ -78,7 +83,7 @@ def __init__(self, loop, host, port: int, worker_id: str, request_id: str,
             name='grpc-thread', target=self.__poll_grpc)
 
     @staticmethod
-    def load_bindings():
+    def load_bindings() -> Dict[Any, Any]:
         """Load out-of-tree binding implementations."""
         services = {}
```

Comment:
Hmm, this is very interesting, seems like **services** is always **an empty dictionary**.

Reply:
this is weird - it's not loading anything and not used anywhere - even interesting, `azure.functions.bindings` does not exist! :)

I am removing this dead code.

Reply:
```python
from azure.functions import bindings
Traceback (most recent call last):
  File "<input>", line 1, in <module>
ImportError: cannot import name 'bindings' from 'azure.functions'
```

Reply:
Yea, I think since we move to use generic binding in azure-functions-python-library, we remove the bindings module from there. https://github.com/Azure/azure-functions-python-library/blob/dev/azure/functions/__init__.py

Reply:
Good to know. Send an update with this removed.